run_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+while [[ "$(kubectl get deployment -n kong | grep 0/1 | wc -l)" != 0 ]]; do
+  echo "waiting for Kong to be ready"
+  sleep 10;
+done
+
+KONG_VERSION=$(kubectl exec -n kong -it $(kubectl get pods -n kong | grep kong | head -n 1 | awk '{print $1}') -- kong version | tr -d '[:space:]')
```

Comment:
Why can't you get this from the deployment YAML itself?

Reply:
the deplayment YAML typically is pinned to `latest` isn't it?

Reply:
Then we should change it to be pinned to a specific version.
Every Kong release, we should verify if there are any updates necessary and then bump it up.

If we want to iterate over this and not make part of this PR, that's cool but we should really be avoiding this hack. Up to you to tackle it in this PR or a later one.